itests/org.jboss.tools.openshift.ui.bot.test/pluginCustomization.ini

```diff
@@ -0,0 +1,8 @@
+org.eclipse.ui.workbench/PLUGINS_NOT_ACTIVATED_ON_STARTUP=org.eclipse.equinox.p2.ui.sdk.scheduler;org.eclipse.m2e.discovery;org.eclipse.epp.logging.aeri.core;org.eclipse.epp.logging.aeri.ide
+org.eclipse.mylyn.commons.notifications.ui/notifications.enabled=false
+org.jboss.tools.project.examples/showCheatsheets=never
+org.eclipse.epp.logging.aeri.ui.prefs/action=IGNORE
+org.eclipse.oomph.setup.ui/enable.preference.recorder=false
+org.eclipse.oomph.setup.ui/skip.startup.tasks=true
+org.jboss.tools.jst.web.ui/Use\ visual\ editor\ for\ html5\ editing=true
+org.jboss.tools.jst.web.ui/Remember\ visual\ editor\ engine=true
```

Comment:
This file should be removed. Using the ITests profile from parent pom will download that file into ${requirementsDirectory}